Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -379,6 +382,8 @@ void function PostDeathThread_MP( entity player, var damageInfo ) // based on ga
 		SetRespawnAvailable( player )
 		
 		wait respawnDelay
+
+		thread ForceRespawnMeSignalAfterDelay( player )
```

Comment:
maybe there should a playlist var to disable this?

Reply:
yeah maybe, idk

Reply:
Open to any suggestions btw, would probably default to being enabled but idk about names and stuff

Reply:
`ns_force_respawn`?

Reply:
Kinda forgor about this PR, but yeah thats a good one